articles/service-bus-messaging/service-bus-performance-improvements.md

```diff
@@ -323,7 +323,7 @@ There are some challenges with having a greedy approach, that is, keeping the pr
 
 ## Multiple queues or topics
 
-If a single queue or topic can't handle the expected, use multiple messaging entities. When using multiple entities, create a dedicated client for each entity, instead of using the same client for all entities.
+If a single queue or topic can't handle the expected amount of messages, use multiple messaging entities. When using multiple entities, create a dedicated client for each entity, instead of using the same client for all entities.
```

Comment:
```suggestion
If a single queue or topic can't handle the expected number of messages, use multiple messaging entities. When using multiple entities, create a dedicated client for each entity, instead of using the same client for all entities.
```